postgresql-11.5/src/fe_utils/printbteq.c

```diff
@@ -923,19 +956,50 @@ print_aligned_text_bteq(const printTableContentBteq *cont, FILE *fout, bool is_p
                     struct lineptr *this_line = col_lineptrs[i] + curr_nl_line;
                     unsigned int nbspace;
 
+                    if (cont->table_width) {
+                        int border_c = (i+1 != cont->ncolumns ? 1 : 0);
```

Comment:
be careful with spaces, new lines and alignments.

Reply:
done as specified

Reply:
`i+1` should be `i + 1` and the same with the other ones:
`width_wrap[i]+2`
etc.

Reply:
done as specified

Reply:
As I can see, it is not done.
Please, be more careful, and if you don't understand what is required/said in a comment, let me know.
I expected to see
`int border_c = (i+1 != cont->ncolumns ? 1 : 0);`
changed into
`int border_c = (i + 1 != cont->ncolumns ? 1 : 0);`